src/common/options/global.yaml.in

```diff
@@ -5028,7 +5028,20 @@ options:
     be transformed to the currrently default 'per-pg' format when BlueStore's quick-fix or
     repair are applied.
   default: false
-  with_legacy: true
+  with_legacy: false
+  see_also:
+  - bluestore_debug_legacy_omap_per_pool
+- name: bluestore_debug_legacy_omap_per_pool
```

Comment:
Wouldn't it be better to have a single bluestore_debug_legacy_omap parameter which is selection between bulk, per_pool and off modes?

Reply:
Done, but restricted to "bulk" "per_pool" since it is "legacy" option.